src/lando/utils/management/commands/setup_dev.py

```diff
@@ -19,28 +21,35 @@ def setup_workers(self):
             SCM_TYPE_HG: None,
         }
 
-        for worker_scm in workers:
+        worker_types = [
+            "",  # landing workers
+            "-automation-worker",
+        ]
+
+        for worker_scm, worker_type in itertools.product(workers, worker_types):
+            worker_name = f"{worker_scm}{worker_type}"
             try:
-                worker = Worker.objects.get(name=worker_scm)
+                worker = Worker.objects.get(name=worker_name)
                 self.stdout.write(f"Found {worker} worker.")
             except Worker.DoesNotExist:
                 # Set the name of the worker to match the SCM.
-                worker = Worker(name=worker_scm, scm=worker_scm)
+                worker = Worker(name=worker_name, scm=worker_scm)
                 worker.save()
                 self.stdout.write(f"Created {worker} worker.")
             finally:
-                workers[worker_scm] = worker
+                workers[worker_name] = worker
 
         for repo in Repo.objects.all():
             # Associate all repos with applicable worker.
-            self.stdout.write(
-                f"Adding {repo} ({repo.scm_type}) to {workers[repo.scm_type]}."
-            )
-            workers[repo.scm_type].applicable_repos.add(repo)
+            for worker in workers.values():
+                if not worker or worker.scm != repo.scm_type:
```

Comment:
What's the `not worker` check for? Presumably, `workers` contains non-null values so this feel redundant.

Reply:
This was to pacify the static analyser, as `workers` is initialised with `None` values at first https://mozilla.slack.com/archives/C4D3JFF26/p1748999453647419

But by the time we're here, it should no longer be the case...

I think I need to change the way `workers` is first declared (: